README.md

```diff
@@ -296,3 +311,436 @@ callback function when an error occurs with navigator.geolocation.
   - Returned when the device is unable to retrieve a position. In general, this means the device is not connected to a network or can't get a satellite fix.
 - `PositionError.TIMEOUT`
   - Returned when the device is unable to retrieve a position within the time specified by the `timeout` included in `geolocationOptions`. When used with `navigator.geolocation.watchPosition`, this error could be repeatedly passed to the `geolocationError` callback every `timeout` milliseconds.
+
+
+## Sample: Get the weather, find stores, and see photos of things nearby with Geolocation ##
+
+Use this plugin to help users find things near them such as Groupon deals, houses for sale, movies playing, sports and entertainment events and more.
+
+Here's a "cookbook" of ideas to get you started. In the snippets below, we'll show you some basic ways to add these features to your app.
+
+* Get your coordinates.
```

Comment:
That's what I was thinking as well. Unfortunately the tags used to create anchor links are ugly and only benefit the HTML view of this file. Folks who read the readme right out of their project will get the text representation of the anchor link - which is like <a id ...> etc. I opted for not littering the top with links as that does not seem to be the protocol being used for other sections of the readme.